sdk/keyvault/keyvault-admin/package.json

```diff
@@ -42,7 +42,7 @@
     "execute:samples": "dev-tool samples run samples-dev",
     "extract-api": "dev-tool run build-package && dev-tool run extract-api",
     "format": "dev-tool run vendored prettier --write --config ../../../.prettierrc.json --ignore-path ../../../.prettierignore \"src/**/*.ts\" \"test/**/*.ts\" \"samples-dev/**/*.ts\" \"*.{js,json}\"",
-    "generate:client": "node ../generate.js",
+    "generate:client": "tsp-client update -d -o src/generated --emitter-options=\"generate-metadata=false;generate-test=false\"",
```

Comment:
do you think these options can go in the specs repo tspconfig.yaml? it would be nice (and I suspect good from an automation point of view) if `tsp-client update` without arguments also Just Worked™

e.g. I see `emitter-output-dir` being used by Go here https://github.com/Azure/azure-rest-api-specs/blob/main/specification/keyvault/Security.KeyVault.Certificates/tspconfig.yaml#L57 and presumably we can update generate-metadata also

Reply:
Ya, good callout - thanks! I'll do that separately and created https://github.com/Azure/azure-sdk-for-js/issues/35071 so I don't forget